src/reconcile_common.ts

```diff
@@ -65,6 +66,9 @@ export default async (recon?: ReconcilerData) => {
    const [reconJpath, jpath] = handleEphemeral(recon.jpath ?? [], core.getMultilineInput('jpath'), false, s => `-J${s}`);
    recon.jpath = reconJpath;
 
+   const [reconInputModules, inputModules] = handleEphemeral(recon.inputModules ?? [], core.getMultilineInput('input-modules'), true, s => `--input-modules=${s}`);
```

Comment:
Хотя подожди, ещё одно перед мержом:
```suggestion
   const [reconInputModules, inputModules] = handleEphemeral(recon.inputModules ?? [], core.getMultilineInput('input-modules'), false, s => `--input-modules=${s}`);
```

Reply:
Иначе синтаксис будет отличный от inputs